build_projects/dotnet-cli-build/DotNetMSBuildTool.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+using Microsoft.DotNet.Cli.Build.Framework;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public abstract class DotNetMSBuildTool : DotNetTool
+    {
+        public DotNetMSBuildTool()
```

Comment:
This constructor is unnecessary. Generally, abstract classes want protected constructors if any but in this case I'd just remove it.